src/main/java/teammates/ui/template/CourseEditInstructorPanel.java

```diff
@@ -18,6 +18,7 @@
     private ElementTag resendInviteButton;
     private ElementTag editButton;
     private ElementTag cancelButton;
+    private ElementTag addInstructorCancelButton;
```

Comment:
This variable name feels abit weird. It suggests the `add` action over the `cancel` action. Should it be `cancelAddInstructorButton`